modules/cluster-logging-collector-log-forwarding-supported-plugins.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-log-forwarding-supported-plugins_{context}"]
+
+= Supported plugins for forwarding logs to third-party systems
+
+[role="_abstract"]
+Red Hat OpenShift Logging version 5.0 tests and supports the following plugins for forwarding logs to third-party systems.
+
+.Plugins
+[options="header"]
+|====
+|Output type|Plugin name|Plugin version|Tested with
+
+| Elasticsearch | link:https://github.com/uken/fluent-plugin-elasticsearch[fluent-plugin-elasticsearch] | link:https://github.com/uken/fluent-plugin-elasticsearch/releases/tag/v4.1.1[4.1.1] | link:https://github.com/openshift/cluster-logging-operator/blob/release-5.0/test/helpers/elasticsearch.go#L219[Elasticsearch 6.8.1]
+
+| Fluent forward protocol | link:https://github.com/fluent/fluentd[fluentd] | link:https://github.com/fluent/fluentd/releases/tag/v1.7.4[1.7.4] | link:https://github.com/openshift/cluster-logging-operator/blob/release-5.0/test/helpers/fluentd/receiver.go#L96[fluent forward v1]
```

Comment:
IMHO, we should change the "Tested with" cell to "Fluentd X.Y" (don't know what version we install to test the forward protocol but maybe @anpingli and @jcantrill can help.

The important part is to highlight the technology not the protocol.

Reply:
@jcantrill @vimalk78  Are you on board with this change. Should the right-most cell state "Fluentd X.Y"? If so, which values should we show for X.Y?

Reply:
@sichvoge Isn't that contrary to what we discussed previously?  cc @alanconway ?  Protocols are an API standard that vendors build against meaning if the standard is implemented correctly then all consumers should be compatible with each other.  IMO its more accurate to state we tested with fluent forward v1 then it is to say fluent v1.7.4

Reply:
It's also important to call out the *fluent forward v1* protocol to differentiate it from *fluent forward v0*, which we don't test.

Reply:
I'll leave this unchanged for now.

Reply:
Can we at least say that we tested it with a Fluentd on the other side since other log aggregator also understand the forward protocol and the current text is to "open"?

Reply:
Maybe we can say something like "fluent forward v1 together with another Fluentd receiver"

Reply:
- The _protocol_ is the important thing to advertise at the forwarder - the same protocol can be used to forward to different endpoints. 
- _However_ we should state the _technology and version_ of the endpoint we _tested with_ as a concrete reference point that we know to work. 
- The technology we use internally (e.g. the internal version of fluentd or the name of  plugins we use) should not appear here, that's private detail that we might change as long as it doesn't change the protocol and the range of endpoints we can forward to. 
Does that make sense?

Reply:
So the colums are: 
| Output Type            | Protocol               | Tested With              |
| fluentd-forward        | fluent forward v1 | fluentd x.y endpoint  | 

I suggest using the same output type names as used in the spec.